proto/ibc/applications/fee/v1/tx.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package ibc.applications.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Msg defines the ibc/fee Msg service.
+service Msg {
+  // RegisterCounterpartyAddress defines a rpc handler method for MsgRegisterCounterpartyAddress
+  // RegisterCounterpartyAddress is called by the relayer on each channelEnd and allows them to specify their
+  // counterparty address before relaying This ensures they will be properly compensated for forward relaying since
+  // destination chain must send back relayer's source address (counterparty address) in acknowledgement This function
+  // may be called more than once by a relayer, in which case, latest counterparty address is always used.
+  rpc RegisterCounterPartyAddress(MsgRegisterCounterpartyAddress) returns (MsgRegisterCounterPartyAddressResponse);
+  // EscrowPacketFee defines a rpc handler method for MsgEscrowPacketFee
+  // EscrowPacketFee is an open callback that may be called by any module/user that wishes to escrow funds in order to
+  // incentivize the relaying of the given packet.
+  rpc EscrowPacketFee(MsgEscrowPacketFee) returns (MsgEscrowPacketFeeResponse);
+}
+
+// MsgRegisterCounterpartyAddress is the request type for registering the counter party address
+message MsgRegisterCounterpartyAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string address              = 1;
+  string counterparty_address = 2 [(gogoproto.moretags) = "yaml:\"counterparty_address\""];
+}
+
+// MsgRegisterCounterPartyAddressResponse defines the Msg/RegisterCounteryPartyAddress response type
+message MsgRegisterCounterPartyAddressResponse {}
+
+// MsgEscrowPacketFee defines the request type EscrowPacketFee RPC
+message MsgEscrowPacketFee {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  ibc.core.channel.v1.PacketId packet_id   = 1 [(gogoproto.moretags) = "yaml:\"packet_id\""];
+  ibc.applications.fee.v1.Fee  receive_fee = 2 [(gogoproto.moretags) = "yaml:\"receive_fee\""];
+  ibc.applications.fee.v1.Fee  ack_fee     = 3 [(gogoproto.moretags) = "yaml:\"ack_fee\""];
+  ibc.applications.fee.v1.Fee  timeout_fee = 4 [(gogoproto.moretags) = "yaml:\"timeout_fee\""];
+  repeated string              relayers    = 5;
```

Comment:
This should use `IdentifiedPacketFee`. Any reason not to?

Reply:
None think it was just outdated